helper/schema/schema_test.go

```diff
@@ -8209,48 +8209,26 @@ func TestValidateAtLeastOneOfAttributes(t *testing.T) {
 	}
 }
 
-func Test_panicOnErrDefaultTrue(t *testing.T) {
-	oldEnv := os.Getenv(PanicOnErr)
+func TestPanicOnErrorDefaultsFalse(t *testing.T) {
+	oldEnv := os.Getenv("TF_ACC")
 
-	os.Setenv(PanicOnErr, "")
-	if !schemaMap(nil).panicOnError() {
-		t.Fatalf("Empty %s should default to true", PanicOnErr)
-	}
-
-	os.Setenv(PanicOnErr, oldEnv)
-}
-
-func Test_panicOnErrParsableTrue(t *testing.T) {
-	oldEnv := os.Getenv(PanicOnErr)
-
-	os.Setenv(PanicOnErr, "true")
-	if !schemaMap(nil).panicOnError() {
-		t.Fatalf("Parsable truthy %s should return true", PanicOnErr)
-	}
-
-	os.Setenv(PanicOnErr, oldEnv)
-}
-
-func Test_panicOnErrParsableFalse(t *testing.T) {
-	oldEnv := os.Getenv(PanicOnErr)
-
-	os.Setenv(PanicOnErr, "false")
+	os.Setenv("TF_ACC", "")
```

Comment:
oh god this is just asking for a race condition problem 😭

Reply:
Agreed, these unit tests aren't super high value so we could ditch the tests if you find this too "anti pattern"